.github/workflows/container-autoupdate.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, test and push container
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  push-master-container:
+    name: Refresh Rawhide container
```

Comment:
I'd avoid putting "Rawhide" here. It doesn't coincide with how the name/tag is going to be (`:master` or `:f33` etc.), and it's a source of diff noise between branches. Let's just say "anaconda-ci container"?

Reply:
kk will do